Makefile

```diff
@@ -182,3 +182,6 @@ vendor:
 		$(GO) mod tidy && \
 		$(GO) mod vendor && \
 		$(GO) mod verify
+
+vendor-in-container:
+	podman run --privileged --rm --env HOME=/root -v `pwd`:/src -w /src docker.io/library/golang:1.13 make vendor
```

Comment:
Might be good at some point to set a variable at the top of the file to `docker.io/library/golang:1.13` so that it can be found/changed more eaily when the next version comes out.